2023/day07/README.md

```
@@ -1,5 +1,5 @@
 # Day 7 Task: Understanding package manager and systemctl
-
+#Please Do the task properly
```

markdownlint-cli2 (0.18.1)

No space after hash on atx style heading

(MD018, no-missing-space-atx)